keras_cv/models/csp_darknet.py

```diff
@@ -36,8 +36,20 @@
 def CSPDarkNet(
     include_rescaling,
     include_top,
-    depth_multiplier=1.0,
-    width_multiplier=1.0,
+    depth_multiplier = {
```

Comment:
The `depth_multiplier` and `width_multiplier` are used to multiply the base depth/width. Having these be dictionaries wouldn't work. Additionally, if we pass a string as the argument for the size - they'd override the `dict` type and fail because of `str`*`float` multiplication.

Please follow the existing convention of exposing different method signatures as the API for the different model configurations, as can be seen with other `keras_cv/models`.

Reply:
thanks for informing

Reply:
For reference, this is what I mean: https://github.com/keras-team/keras-cv/blob/df7a49528b051efff826bd4fd30b9ce468c405f1/keras_cv/models/resnet_v2.py#L29